docs/website/binding_types_reference_guide.md

```diff
@@ -2083,6 +2083,52 @@ Example:
 You can then call the extension method `GetDomain` to get the domain constant of
 any error.
 
+
+## BindAsAttribute
+
+The `BindAsAttribute` allows binding `NSNumber` `NSValue` and `NSString`(enums) into more accurate C# types this allows an easy way to create better .NET API.
```

Comment:
In fact this section it very similar to the one above.

Here you should describe the attribute (reference, not how-to, guide), i.e. 
* where it applies (return and parameter types);
* which types can be used (input and output);